packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -128,7 +118,6 @@ const AccountTransferForm = observer(
         const [to_accounts, setToAccounts] = React.useState({});
         const [transfer_to_hint, setTransferToHint] = React.useState<string>();
 
-        const is_from_pre_appstore = is_pre_appstore && !location.pathname.startsWith(routes.cashier);
```

Comment:
Why remove this? 
`
const is_from_outside_cashier =  !location.pathname.startsWith(routes.cashier);
`

Reply:
Will re-add this backl